AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -6145,6 +6145,9 @@ afyonhaber.com,tamindir.com,bigfishgames.de,adapostasi.com##div[id^="simplemodal
 temaretik.com##div[ng-if="$root.joinGuest"]
 techblogng.com##iframe.HB-Takeover[src="about:blank"]
 com-magazin.de##w-div[role$="dialog"]
+circleofcricket.com#$#body { overflow: visible!important; }
+circleofcricket.com#$#.modal-backdrop { display: none!important; }
+circleofcricket.com#$##myModal { display: none!important; }
```

Comment:
This popup looks like an ad (it redirects to third-party website when clicked), so could you please move it to - https://github.com/AdguardTeam/AdguardFilters/blob/master/BaseFilter/sections/specific.txt

Also please add spaces before `!important`, like:
```adblock
circleofcricket.com#$#body { overflow: visible !important; }
circleofcricket.com#$#.modal-backdrop { display: none !important; }
circleofcricket.com#$##myModal { display: none !important; }
```
It's not an issue, but as far as I know, we prefer to add it this way.